api/src/main/java/org/openmrs/DrugReferenceMap.java

```diff
@@ -12,45 +12,74 @@
 import java.io.Serializable;
 import java.util.Date;
 
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.Parameter;
+import org.hibernate.annotations.GenericGenerator;
 import org.hibernate.envers.Audited;
 import org.hibernate.search.annotations.ContainedIn;
 import org.hibernate.search.annotations.DocumentId;
 import org.hibernate.search.annotations.IndexedEmbedded;
+import org.hibernate.annotations.CascadeType;
+
+import javax.persistence.*;
```

Comment:
please set up your IDE with openmrs, https://openmrs.atlassian.net/wiki/spaces/Archives/pages/25506949/How-To+Setup+And+Use+Your+IDE

we do not use wildcard imports

Reply:
I finished setting up the IDE according to the documentation and resolved the above mentioned wildcard import.